lib/jellyfish/room.ex

```diff
@@ -436,23 +442,28 @@ defmodule Jellyfish.Room do
   end
 
   @impl true
-  def handle_info(
-        %EndpointAdded{endpoint_id: endpoint_id},
-        state
-      )
+  def handle_info(%EndpointAdded{endpoint_id: endpoint_id}, state)
       when endpoint_exists?(state, endpoint_id) do
     {:noreply, state}
   end
 
   @impl true
-  def handle_info(%TrackAdded{} = track_info, state) do
-    Logger.info("Endpoint #{track_info.endpoint_id} added track #{inspect(track_info)}")
+  def handle_info(%TrackAdded{endpoint_id: endpoint_id} = track_info, state)
+      when endpoint_exists?(state, endpoint_id) do
```

Comment:
Shouldn't we handle a situation when we receive a `TrackAdded` notification for an endpoint that doesn't exist? For example, we could raise an error with a proper message.